samples/subsys/ipc/ipc_service/icmsg/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+#
```

Comment:
I think @emob-nordic wanted to modify the original sample but was asked to add a separate sample :)
The value the change carries is showing the bandwidth limits. We can add this code later on either by modifying the single ipc_service sample or adding a bandwidth checking specific test.

@emob-nordic If this is a controversy I propose to drop the change related to creating a sample.

Reply:
Probably my idea is not possible with Kconfig, because ipc_service backend is configured with dt. The sample would need to contain dt overrides for each backend, and potentially overrides for boards or socs. Not sure if it would be feasible to maintain. What do you think?

Reply:
My main goal of this PR is to add backend. I don't want to block this PR because sample is too controversial. I will remove the commit that adds the sample and add it in separate PR, would this be okay @hubertmis  @carlocaione ?.

Reply:
+ i will remove the sample commit and add unit tests for `ipc_icmsg_buf`.

Reply:
I think it's find to create a sample like the one you've already created. We can merge them later if we find it useful.
I prefer to have a separated sample than to have none. Without a sample it's hard to test changes.

Reply:
agree. Please leave the sample in place as a different one from the one testing the other backend.

Reply:
Will bring it back then.